### DISCUSSION TITLE
Using keda helm chart from kedacore but the scaling is not taking place:

### DISCUSSION THREAD

**Discussion by sayan-torch** [kedacore/keda#5894] [General]
Hi team, need a help
I have installed the helm chart for KEDA, and alongwith it the crd's have also been installed
I am deploying keda from makefile 
keda: .kube/config
	${HELM} repo add kedacore https://kedacore.github.io/charts && \
	${HELM} repo update && \
	${HELM} upgrade --install keda kedacore/keda \
	--version 2.14.2 \
	--namespace kube-system \
	--set "scaledObjects.spec.scaleTargetRef.apiVersion=apps/v1" \
	--set "scaledObjects.spec.scaleTargetRef.kind=Deployment" \
	--set "scaledObjects.spec.scaleTargetRef.name=deployment-name" \
	--set "scaledObjects.spec.minReplicaCount=1" \
	--set "scaledObjects.spec.maxReplicaCount=10" \
	--set "scaledObjects.spec.cooldownPeriod=30" \
	--set "scaledObjects.spec.pollingInterval=15" \
	--set "scaledObjects.spec.triggers.type=http" \
	--set "scaledObjects.spec.triggers.metadata.url=http://service-name.kube-system.svc.cluster.local:5000/metrics" \
	--set "scaledObjects.spec.triggers.metadata.method=GET" \
	--set "scaledObjects.spec.triggers.metadata.valueLocation=/job_latency" \
	--set "scaledObjects.spec.triggers.metadata.targetValue=50"
	
But KEDA s=is not scaling my flask application although the port and the metrics endpoint have been correctly exposed which has been tested. Also, i see no scaledobjects.keda.sh is deployed in my namespace. I am confused as am I supposed to deploy this scaledobject manually by writing a yaml file, but if i do so the crd gets conflicted and the scledobject crd is not installed automatically.

The 3 keda pods are running proeprly without any errors but the connection to this service is not getting scaled, any idea?

**Comment by zroubalik** [comment]
Hi, most likely you don't create the ScaledObject correctly. The combination of a Makefile and Helm is a bit confusing to me.
